docs/how-to/how-to-build-and-test-ccv0.md

```diff
@@ -333,6 +333,11 @@ to create containers from these images using `crictl`:
 the guest agent code has been implemented, the Kata confidential-containers build needs to be run with 
 `export SKOPEO=yes` as documented [above](#basic-script-set-up-and-optional-environment-variables) in order to use the
 image signature verification function.
+
+- The policy file, along with the public GPG key and signatures file, will need to be copied over before the pod is created 
+by using the `copy_files_to_guest` function in ccv0.sh. 
+To do this, you should run `~/ccv0.sh copy_files_to_guest` before creating the sandbox pod in the steps below.
```

Comment:
Given that the next instruction is 'If you don't already have a Kata sandbox pod created with `crictl`, then follow the'..., do we need to update it to get the user to create the pod even if they had one previously, if the `copy_files_to_guest` command it needed before the pod is created?

Reply:
I hadn't thought of that, thank you! I've added details on what to do if you **do** have a pod and what to if you **don't**.

Reply:
So I did a manual test of running `~/ccv0.sh copy_files_to_guest` when I already had a running pod and then doing `~/ccv0.sh crictl_create_cc_pod` after and that worked fine, so I think we can simplify the instructions to something like:
- Leave the old instruction about creating a crictl pod linking above
- Add a new instruction to say the user needs to use copy_files_to_guest to enable the guest image and then re-run `~/ccv0.sh crictl_create_cc_pod` to create a pod from that image
How does that sound?

Reply:
That's a good point actually because the create pod function deletes the existing one so we don't need to run the manual delete.
I've updated the instructions to make them simpler as suggested 🙂